### PR TITLE
The continuing of rspec integration.

### PR DESCRIPTION
**Pull request by sdwolf**
Will contain mostly code cleanup and tests.

### CODE REVIEW

**Comment by sdwolf** [comment]
OK i think I've done enough damage for a single PR. Any thoughts on changes so far?

**Comment by ixti** [comment]
LGTM. :+1:

File: lib/redmine_tags/patches/wiki_controller_patch.rb [hunk 1]
```diff
@@ -1,3 +1,21 @@
+# This file is a part of redmine_tags
```
Comment (sdwolf) [inline lib/redmine_tags/patches/wiki_controller_patch.rb]: I wonder, should this license header be included in all files?
What I understood from my google searches is that a LICENSE file in the root folder is enough (at least for FOSS).
Also some projects (like [elixir](https://github.com/elixir-lang/elixir)) do not include it.
I also read stories about needing to update the Copyright year with a huge commit at the beginning of the year.
I am not saying to remove them from all files, but maybe not include them in new ones? @ixti what do you think?

Comment (ixti) [inline lib/redmine_tags/patches/wiki_controller_patch.rb]: I'm OK to remove if from headings of all files as well as simply skip it for new ones only...
In fact I was just following [GPL's How To](https://www.gnu.org/licenses/gpl-howto.html).
And I saw this practice in some other products... But I am absolutely agree that updating
year each time will be creating stupid huge commit... Pretty useless one.

After all, again, I'm absolutely fine to remove that heading from each file.
LICENSE alone is more than enough I believe.

Also, in fact Copyright line became pretty outdated ;))
Probably we should change it to:

```
Copyright (c) 2010 Alexey Zapparov
Copyright (c) 2011-2016 Redmine Tags contributors
```

And generate CONTRIBUTORS file upon releases.

**Pull request merged**